react/components/EditorContainer/Topbar/index.tsx

```diff
@@ -47,7 +46,9 @@ const Topbar: React.FunctionComponent<Props> = ({
     onChangeUrlPath(url)
   }
 
-  const handleUrlKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+  const handleUrlKeyPress: (
```

Comment:
⛏  You can declare the function type on the definition:
```
  const handleUrlKeyPress = (e: React.KeyboardEvent<HTMLInputElement>): void => {
    onEnter(e, () => {
      handleNavigateToUrl
    })
  }
```

You can also remove the closure:
```
  const handleUrlKeyPress = (e: React.KeyboardEvent<HTMLInputElement>): void => {
    onEnter(e, handleNavigateToUrl)
  }
```

Reply:
Nice, I was the one that suggested he did that way, but yours is definitely better :)

Reply:
Nice one! Thanks for the example